File: python/Programmers_level/level1/소수찾기.py
def solution(n):
    def isprime(a):
        if a == 1:
            return False
        if a == 2:
            return True
        for x in range(2, int(a**0.5) + 1):
            if a % x == 0:
                return False
        return True
    
    result = 0
    for i in range(1, n+1):
        if isprime(i):
            result += 1
    
    return result

File: Javascript/Programmers_level/level1/소수 찾기.js
function solution(n) {
  var answer = 0;
  const isPrime = (n) => {
    if (n === 2) return true;
    else if (n === 3) return true;
    else {
      for (let i = 2; i < Math.sqrt(n) + 1; i++) {
        if (n % i === 0) return false;
      }
      return true;
    }
  };

  for (let j = 2; j < n + 1; j++) {
    if (isPrime(j)) {
      answer += 1;
    }
  }

  return answer;
}
